scripts/v6/stored-procs/07-export-solar-scoa-journal-rollup.sql

```diff
@@ -73,15 +73,15 @@ BEGIN
 		sj.FinancialField as JOURNAL_DESCRIPTION_2,
 		'' as JOURNAL_DESCRIPTION_3,
 		dbo.convertDateToInt(sj.EffectiveDate) as TRANSACTION_DATE,
-		sj.SCOA_Function as SCOA_FUNCTION_GUID,
-		sj.SCOA_Fund as SCOA_FUND_GUID,
+		sj.SCOA_Function_Credit as SCOA_FUNCTION_GUID,
+		sj.SCOA_Fund_Credit as SCOA_FUND_GUID,
 		sj.SCOA_Item_Credit as SCOA_ITEM_GUID,
-		sj.SCOA_Project as SCOA_PROJECT_GUID,
-		sj.SCOA_Costing as SCOA_COST_GUID,
-		sj.SCOA_Region as SCOA_REGION_GUID,
-		sj.SCOA_Mun_Classification as ENTITY_COST_CENTRE,
+		sj.SCOA_Project_Credit as SCOA_PROJECT_GUID,
+		sj.SCOA_Costing_Credit as SCOA_COST_GUID,
+		sj.SCOA_Region_Credit as SCOA_REGION_GUID,
+		sj.SCOA_Mun_Classification_Credit as ENTITY_COST_CENTRE,
 		sj.ItemBreakdown_Credit as ENTITY_SUB_ITEM,
-		sj.ProjectID as ENTITY_PROJECT,
+		sj.ProjectID_Credit as ENTITY_PROJECT,
```

Comment:
I believe this should be sj.ProjectBreakdown_Credit. ProjectBreakdown is SOLAR's Entity_Project_Code that they expect back.